torch/testing/_internal/common_methods_invocations.py

```diff
@@ -108,7 +123,9 @@ def __init__(self,
                  promotes_integers_to_float=False,  # whether op promotes unary output to float or not
                  sample_inputs_func=None,  # function to generate sample inputs
                  aten_name=None,  # name of the corresponding aten:: operator
-                 supports_sparse=False  # supported for sparse
+                 aliases=None,  # list of aliases, e.g. torch.absolute -> torch.abs
```

Comment:
"list" -> "iterable"

We want to prefer tuples instead of lists, since tuples are immutable, but any iterable will do

Reply:
The comment should give an example of the correct way to add an alias, which would be `("absolute",)` for `torch.abs`, right?